src/v/kafka/server/handlers/fetch.cc

```diff
@@ -146,14 +147,11 @@ read_result::memory_units_t::~memory_units_t() noexcept {
     if (shard == ss::this_shard_id()) {
         return;
     }
-    auto f = ss::smp::submit_to(
+    ssx::background = ss::smp::submit_to(
```

Comment:
This change isn't in the original PR, but it';s a new seastar feature, and this change is fine.